src/modules/Elsa.JavaScript/TypeDefinitions/Providers/CommonFunctionsDefinitionProvider.cs

```diff
@@ -91,6 +91,26 @@ protected override IEnumerable<FunctionDefinition> GetFunctionDefinitions(TypeDe
             .Name("toJson")
             .Parameter("value", "any")
             .ReturnType("string"));
+        
+        yield return CreateFunctionDefinition(builder => builder
+            .Name("bytesToString")
+            .Parameter("value", "Byte[]")
+            .ReturnType("string"));
+        
+        yield return CreateFunctionDefinition(builder => builder
+            .Name("bytesFromString")
+            .Parameter("value", "string")
+            .ReturnType("Byte[]"));
+        
+        yield return CreateFunctionDefinition(builder => builder
+            .Name("bytesToBase64")
+            .Parameter("value", "Byte[]")
+            .ReturnType("string"));
+        
+        yield return CreateFunctionDefinition(builder => builder
+            .Name("bytesFromBase64")
+            .Parameter("value", "string")
+            .ReturnType("Byte[]"));
 
```

Comment:
Should we add methods for converting between Base64 and string directly?

Reply:
Why not indeed, might be convenient 👍🏻